ddtrace/contrib/asyncio/__init__.py

```diff
@@ -19,8 +19,12 @@
     loop.run_until_complete(some_work())
     loop.close()
 
-Many helpers are provided to simplify how the tracing ``Context`` is handled
-between scheduled coroutines and ``Future`` invoked in separated threads:
+If ``contextvars`` is available, we use the ``DefaultContextProvider``,
```

Comment:
we should expand on this a bit, e.g. `ddtrace.context.DefaultContextProvider` otherwise we use the legacy `ddtrace.contrib.asyncio.provider.AsyncioContextProvider`

So people understand what `DefaultContextProvider` actually means.

Also, in Sphinx we can cross-reference classes via:

```rst
:class:`ddtrace.context.DefaultContextProvider`
```

Not required, but FYI